Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM python:3.10-slim-bookworm@sha256:d364435d339ad318ac4c533b9fbe709739f9ba006b0721fd25e8592d0bb857cb
+# syntax=docker/dockerfile:1
+FROM python:3.10-slim
```

Comment:
What was the reason for using a loose tag? I'm starting to think it makes sense as it will allow to get the latest improvements for each release as long as we test the resulting Docker image to avoid surprises.

But still wondering if there was something else.